gap/perm.gi

```diff
@@ -347,24 +347,44 @@ end;
 #! @BeginChunk ThrowAwayFixedPoints
 #! This method defines a homomorphism of a permutation group
 #! <A>G</A> to the action on the moved points of <A>G</A> if
-#! <A>G</A> does not have too many moved points. In the current setup, the
-#! homomorphism is defined if the number <M>k</M> of moved
+#! either <A>G</A> has a lot of fixed points, or if <A>G</A> knows that it is
+#! primitive and has fixed points. If <A>G</A> has fixed points but does not
+#! know whether it is primitive, then it returns <K>NotEnoughInformation</K>
+#! so that it may be called again at a later time.
+#! In all other cases, it returns <K>NeverApplicable</K>.
+#! <P/>
+#!
+#! In the current setup, the
+#! homomorphism is defined if the number <M>n</M> of moved
 #! points is at most <M>1/3</M> of the largest moved point of <A>G</A>,
-#! or  <M>k</M> is at most half of the number of points on which
-#! <A>G</A> is stored internally by &GAP;. The method returns
-#! <K>NeverApplicable</K> if it does not define a homomorphism indicating that it will
-#! never succeed.
+#! or  <M>n</M> is at most half of the number of points on which
+#! <A>G</A> is stored internally by &GAP;.
```

Comment:
Perhaps this should be explained a bit more, but then it *is* a bit technical... Perhaps what I wrote above could be extended... something along these lines:

> The reason we do not just always discard fixed points is that it also incurs a cost, so we only try to do it when it seems worthwhile to do so.
>
> To explain what was meant above by the ration of fixed points to moved points, suppose the group G is
> not known to be primitive. Then we still apply this method if one of the following two criterion is met:
> 1. the largest moved point of G is three or more times larger than the number n of actually moved points 
> 2. there is a generator of G whose internal storage reserves space for a number of moved points which is two or more times larger than n (note that even for a simple transposition (1,2), for technical reasons it can happen that GAP internally stores it with the same size as a permutation moving a million points; this is wasteful, and the second criterion tries to deal with this)

Reply:
I agree that that's a bit technical. So I'll put it into a source comment but not into the manual.